OpenEdXMobile/res/layout/fragment_find_courses.xml

```diff
@@ -16,6 +16,18 @@
         android:listSelector="@android:color/transparent"
         tools:listitem="@layout/row_course_list" />
 
+    <TextView
+        android:id="@+id/no_courses"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textAlignment="center"
+        android:text="@string/no_available_courses"
+        android:textSize="24dp"
+        android:visibility="gone"
+        android:layout_centerInParent="true"
+        android:layout_gravity="center"
+        />
```

Comment:
This can be moved at the end of the line above.

Reply:
not sure exactly what you are referring to here,

Reply:
```xml
        android:layout_gravity="center" />
```

instead of:

```xml
        android:layout_gravity="center"
        />
```